### ISSUE TITLE
Fix german translation

### ISSUE THREAD

**Issue by marcusfellinger** [thomasdilts/cross-connect#9]
crossc_de.xml contains many flawed translations. For example

- "Add a line between notes" should be translated "Zeile zwischen Notizen hinzufügen" instead of "Fügen Sie eine Zeile zwischen den Noten"
- "Custom bible download addresses" should be translated "Benutzerdefinierte Adressen für Bibel-Download" instead of "Benutzerdefinierte bible herunterladen Adressen"

**Comment by danielksan81** [comment]
Hi, can I help you with this?

**Comment by thomasdilts** [comment]
Yes. Do you speak German?  Would be nice to fix all the errors in the file instead of just one or 2 lines.

**Comment by danielksan81** [comment]
Yes, I can have a look.

**Comment by thomasdilts** [comment]
There are actually 3 crossc_de.xml but only 2 of them are actual. They are **/CrossConnect10/CrossConnect10/properties/crossc_de.xml**   and **/CrossConnect8.1/CrossConnectPhone/properties/crossc_de.xml**.    These 2 files are very similar but not identical. Both need to be fixed

**Comment by danielksan81** [comment]
Sure, will have a look.
